hydra/core/override_parser/overrides_visitor.py

```diff
@@ -182,8 +191,6 @@ def visitElement(self, ctx: OverrideParser.ElementContext) -> ParsedElementType:
     def visitValue(
         self, ctx: OverrideParser.ValueContext
     ) -> Union[ChoiceSweep, RangeSweep, IntervalSweep, ParsedElementType]:
-        if self.is_matching_terminal(ctx, "<EOF>"):
-            return ""
```

Comment:
I suspect this is legacy code that wasn't used anymore, because raising an exception here didn't break tests => I just removed it